<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreDashAdminClass</class>
 <widget class="QWidget" name="StoreDashAdminClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StoreDashAdmin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="Ribbon" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="nodeButton">
          <property name="toolTip">
           <string>Edit organization</string>
          </property>
          <property name="text">
           <string>Node</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectionButton">
          <property name="toolTip">
           <string>Set measurement units</string>
          </property>
          <property name="text">
           <string>Connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shelfButton">
          <property name="toolTip">
           <string>Insert shelf</string>
          </property>
          <property name="text">
           <string>Shelf</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="drawingArea" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="DrawingArea" name="graphicsView"/>
        </item>
        <item>
         <widget class="QLabel" name="posLabel">
          <property name="text">
           <string>00 , 00</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawingArea</class>
   <extends>QGraphicsView</extends>
   <header>drawingarea.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="StoreDashAdmin.qrc"/>
 </resources>
 <connections/>
</ui>
